core/src/main/java/org/opensearch/sql/data/model/ExprValueUtils.java

```diff
@@ -15,6 +19,8 @@
 import org.opensearch.sql.data.type.ExprCoreType;
 import org.opensearch.sql.exception.ExpressionEvaluationException;
 
+// TODO merge file from #122
```

Comment:
TODO?

Reply:
#122 is still on review, but published earlier. This file will produce one huge merge conflict, so I planned to rebase this branch after releasing #122.